invoice/signals.py

```diff
@@ -52,3 +68,42 @@ def save_invoice_in_db(**kwargs):
                     invoice_update["amount_total"] += float(result_invoice_line["data"]["amount_total"])
                     invoice_update["amount_discount"] += 0 if result_invoice_line["data"]["discount"] else result_invoice_line["data"]["discount"]
             invoice_service.update(invoice_update)
+
+
+def check_bill_exist(**kwargs):
+    function_arguments = kwargs.get('data')[1]
+    instance = function_arguments.get('instance', None)
+    if instance.__class__.__name__ == "QuerySet":
+        queryset_model = instance.model
+        if queryset_model.__name__ == "Claim":
+            claim = instance.first()
```

Comment:
Is it possible that query will return >1 entry? Maybe it'll be better to use .get() and raise exception if more than one entry is returned.

Reply:
Yes, Could be more than 1. But this is about to check if claim is in BillItem. I can explain this details what I intended to do tomorrow